src/layouts/ModelizerDrawLayout.vue

```diff
@@ -16,6 +16,9 @@
     <q-page-container>
       <modelizer-draw-page />
     </q-page-container>
+
+    <console-footer is-diagram-error />
```

Comment:
I thought that we did not want the console footer on the drawer but only on text ?